backend/src/main/java/com/javahelp/backend/data/DynamoDBUserStore.java

```diff
@@ -334,14 +338,17 @@ private static User userFromDynamo(Map<String, AttributeValue> item) {
                     firstName = item.get("firstName").getS(),
                     lastName = item.get("lastName").getS(),
                     address = item.get("address").getS();
-            info = new ClientUserInfo(email, address, phoneNumber, firstName, lastName);
+            info = new ClientUserInfo(email, address, phoneNumber, firstName, lastName,
+                    Gender.FEMALE);
```

Comment:
I'm not too sure why you specify Gender with FEMALE at here.

Reply:
To fill out the `Gender` parameter for this `Client` :)

Reply:
I also don't understand why you specify female, why not get the gender attribute from the database, parse it to the gender enum, and assign it to the user info?

Reply:
I see